src/scripts/main.js

```diff
@@ -354,7 +354,40 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboardTable = document.querySelector('.dashboard');
```

Comment:
Ensure that the `.dashboard` element exists in the HTML. If it doesn't, `dashboardTable` will be null, and attempting to append rows will result in an error. Consider adding a check to ensure `dashboardTable` is not null before proceeding with the table creation.